android-auto-app/src/main/res/values/themes.xml

```diff
@@ -1,9 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-  <style name="CarAppTheme">
+  <style name="CarAppTheme" parent="@style/Theme.AppCompat.NoActionBar">
     <item name="carColorPrimary">#ff41af6a</item>
     <item name="carColorPrimaryDark">#ff41af6a</item>
     <item name="carColorSecondary">#fff3b713</item>
     <item name="carColorSecondaryDark">#fff3b713</item>
+
+    <item name="colorPrimary">#ff41af6a</item>
```

Comment:
should we remove colors with prefix `car`?

Reply:
Those are needed for the car style, here's a [sample app from the androidx car repo](https://github.com/androidx/androidx/blob/6179f2dea9e30e7d90449e274ec284b1821e2d5e/car/app/app-samples/places/common/src/main/res/values/themes.xml)

And then specify the [car theme in the manifest](https://github.com/androidx/androidx/blob/75784ce6dbac6faa5320e5898e9472f02ab8710c/car/app/app-samples/showcase/mobile/src/main/AndroidManifestWithSdkVersion.xml#L59-L62)

I separated the themes to remove confusion though. Seems good to have a `AppTheme` and a `CarTheme`